.travis.yml

```diff
@@ -75,20 +75,39 @@ script:
     - vendor/bin/phpunit test/integrationtests/
 
     # Run integration tests for specific modules
-    - vendor/bin/phpunit modules/mri_upload/test/mri_uploadTest.php
+    - vendor/bin/phpunit modules/mri_upload/test/
     - vendor/bin/phpunit modules/statistics/tests/
-    - vendor/bin/phpunit modules/dashboard/test/DashboardTest.php
-    - vendor/bin/phpunit modules/configuration/test/ConfigurationTest.php
-    - vendor/bin/phpunit modules/user_accounts/test/user_accountsTest.php
-    - vendor/bin/phpunit modules/timepoint_flag/test/timepoint_flagTest.php
-    - vendor/bin/phpunit modules/survey_accounts/test/survey_accountsTest.php
-    - vendor/bin/phpunit modules/reliability/test/reliabilityTest.php
-    - vendor/bin/phpunit modules/next_stage/test/next_stageTest.php
-    - vendor/bin/phpunit modules/mri_violations/test/mri_violationsTest.php
-    - vendor/bin/phpunit modules/mri_upload/test/mri_uploadTest.php
-    - vendor/bin/phpunit modules/instrument_manager/tests/instrument_managerTest.php
-    - vendor/bin/phpunit modules/instrument_list/tests/instrument_listTest.php
-    - vendor/bin/phpunit modules/instrument_builder/test/instrument_builderTest.php
+    - vendor/bin/phpunit modules/dashboard/test/
+    - vendor/bin/phpunit modules/configuration/test/
+    - vendor/bin/phpunit modules/user_accounts/test/
+    - vendor/bin/phpunit modules/timepoint_flag/test/
+    - vendor/bin/phpunit modules/survey_accounts/test/
+    - vendor/bin/phpunit modules/reliability/test/
+    - vendor/bin/phpunit modules/next_stage/test/
+    - vendor/bin/phpunit modules/mri_violations/test/
+    - vendor/bin/phpunit modules/mri_upload/test/
+    - vendor/bin/phpunit modules/instrument_manager/tests/
+    - vendor/bin/phpunit modules/instrument_list/tests/
+    - vendor/bin/phpunit modules/instrument_builder/test/
+
+    - vendor/bin/phpunit modules/brainbrowser/test
+    - vendor/bin/phpunit modules/candidate_list/test
+    - vendor/bin/phpunit modules/candidate_parameters/test
+    - vendor/bin/phpunit modules/certification/test
+    - vendor/bin/phpunit modules/configuration/test
+    - vendor/bin/phpunit modules/conflict_resolver/test
+    - vendor/bin/phpunit modules/create_timepoint/test
+    - vendor/bin/phpunit modules/data_integrity_flag/test
```

Comment:
The data integrity file test is in: modules/data_integrity_flag/tests/data_integrity_flagTest.php in this pull request, (not test), so this is causing it to fail.